src/collections/members/naimat-oyewale/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+name: Naimat Oyewale
+position: Contributor
+image_path: ./naimat-oyewale.jpeg
+github: Naimaoye
+twitter: Naimatoyewale
+linkedin: Naimot Oyewale
```

Comment:
This id seems to be broken. @Naimahtech Can you please provide the necessary information so that @Blakelist7 can update it.

Reply:
Thanks @Jashpatel1. LinkedIn profile: https://www.linkedin.com/in/naimot-oyewale-828a88154/

Reply:
Thanks